### PR TITLE
STRWN Codechef Challenge in C++

### PR DESCRIPTION
**Pull request by Kuljeet-123**
**Fixes issue:** #4550 

**Changes:**

added a solution of a codechef challenge [STRWN](https://www.codechef.com/COMR2019/problems/STRWN) in C++.

### CODE REVIEW

**Comment by Kuljeet-123** [comment]
@arnavb Please review it and merge it also.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp:2]:
```suggestion
```

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#include <cstring>
+
+using namespace std;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp:5]: Don't use this.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+#define MOD 1000000007
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp:7]: Use `constexpr` instead.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+uint64_t modmul(const uint64_t x, const uint64_t y)
+{
+	if (x > (1 << 30) && y > (1 << 30))
+		return ((x >> 30)*((y << 30) % MOD) + y*(x & ((1 << 30) - 1))) % MOD;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp:12]: Reformat this file using clang format with 4 spaces.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+	while (t-- > 0)
+	{
+		int n;
+		cin >> n;
+		uint64_t ans = modpow(3, n);
+		if (n % 2 == 0)
+			ans = (ans + 3) % MOD;
+		else
+			ans = (ans + MOD - 3) % MOD;
+		cout << ans << endl;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp:48]: Use `\n` not `endl`.

File: code/online_challenges/src/codechef/STRWN/STRWN.cpp [hunk 6]
```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+
+#define ll long long
+#define ld long double
+#define pb push_back
+#define pp pop_back
+#define mp make_pair
+#define ff first
+#define ss second
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STRWN/STRWN.cpp:9]: Don't use any of these.

File: code/online_challenges/src/codechef/STRWN/STRWN.cpp [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+#define maxn 1000000007
+#define PI 3.14159265358979323846
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STRWN/STRWN.cpp:11]: Should be constexpr.

File: code/online_challenges/src/codechef/STRWN/STRWN.cpp [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+using namespace std;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STRWN/STRWN.cpp:13]: Don't use this.

File: code/online_challenges/src/codechef/STRWN/STRWN.cpp [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+#define maxn 1000000007
+#define PI 3.14159265358979323846
+
+using namespace std; 
+
+int main()
+{
+    ios_base::sync_with_stdio(false);
+    cin.tie(NULL);
+    //cout<<fixed<<setprecision(20);
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STRWN/STRWN.cpp:19]: Reformat with clang format and 4 spaces indent.

File: code/online_challenges/src/codechef/STRWN/STRWN.cpp [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    ll a,b,c;
+    cin>>a>>b>>c;
+    if(a==0 && b==0 && c==0)
+    {
+        cout<<0<<endl;
+        return 0;
+    }
+    if(a==0 && b==0 && c>0)
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STRWN/STRWN.cpp:27]: Instead of all these return 0's, use else if.

**Comment by Kuljeet-123** [comment]
@arnavb Due to conflicts , I am closing this PR and will raise a new PR. Because I  forked the repo again

**Pull request closed without merging**